src/routes/v2/siti-agri-schema.oas.yml

```diff
@@ -63,6 +63,91 @@ paths:
                     <html><body><h1>403 Forbidden</h1>
                     Request forbidden by administrative rules.
                     </body></html>
+        '500':
+          description: 'Internal server error.'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  code:
+                    type: integer
+                    example: 500
+                  message:
+                    type: string
+                    example: >
+                      There was an error processing your request. It has been logged (ID 704b9f3b19536a35).'
+  /SitiAgriApi/cv/agreementsByBusiness/sbi/{sbi}/list:
+    get:
+      tags:
+        - 'siti-agri'
+        - 'agreements'
+      summary: 'Find Agreements by SBI.'
+      description: 'Find Agreements by SBI'
+      operationId: 'findAgreementsBySBI'
+      parameters:
+        - name: 'sbi'
+          in: 'path'
+          description: 'sbi'
+          required: true
+          schema:
+            type: 'integer'
+            maximum: 99999999999999999999
+            minimum: 1
+            examples:
+              - '1000000000'
+              - '1111111111'
+              - '2222222222'
+      responses:
+        '200':
+          description: 'Returns full organisation details.'
+          content:
+            application/json:
+              schema:
+                type: 'object'
+                additionalProperties: false
+                properties:
+                  data:
+                    type: 'array'
+                    items:
+                      $ref: '#/components/schemas/AgreementsResponse'
+                  success:
+                    type: 'boolean'
+                  errorString:
+                    type: [string, 'null']
+                    description: 'Error string if present, otherwise can be null'
+                required:
+                  - 'data'
+                  - 'success'
+                  - 'errorString'
```

Comment:
There's a way to reuse a common object wrapper schema like this - the code in `feat/fake-cph` does exactly that